corehq/apps/app_manager/templates/app_manager/partials/case_list_properties.html

```diff
@@ -45,6 +45,12 @@
                     <td class="col-sm-3" data-bind="css: {'has-error': showWarning}">
                         <div data-bind="html: hqImport('app_manager/js/details/screen_config').CC_DETAIL_SCREEN.getFieldHtml(field.val()), visible: !field.edit"></div>
                         <div data-bind="jqueryElement: field.ui, visible: field.edit"></div>
+                        {% if request|toggle_enabled:'USE_XPATH_EXPRESSION' %}
```

Comment:
@nickpell Calculations are behind an [add-on](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/add_ons.py#L81-L85), this should also check that's on - it'd be weird to have this show up for an app that has calculations turned off.